README.md

```diff
@@ -63,7 +63,7 @@ Congratulations, you just wrote an IRC bot!
 
 #### Legobot.legobot()
 
-`Legobot.legobot(host,port,nick,chans, logfunc = "", hostpw = "", defaultFunc = None, defaultFuncChar = "")`
+`Legobot.legobot(host,port,nick,chans, nickpass = "", logfunc = "", hostpw = "", defaultFunc = None, defaultFuncChar = "")`
```

Comment:
We just merged this change in another PR. If we don't have any conflicts, I'll try to leave this part be, but odds are we'll have to drop this part.